src/classes/Fuego.ts

```diff
@@ -11,6 +11,7 @@ export class Fuego {
   public auth: typeof firebase.auth
   public functions: typeof firebase.functions
   public storage: typeof firebase.storage
+  public DocumentReference = firebase.firestore.DocumentReference
```

Comment:
I managed to fix compatibility with Firebase `8.x` by placing the `DocumentReference` inside the Fuego class.
So, for those that use `8.x`, all they need to do is add the `DocumentReference` to their Fuego instance.
It's not a breaking change if they don't add it, because people with a custom Fuego will still be able to use the library like usual, and their document references won't be serialized properly (which it didn't before anyway) until they add `DocumentReference`

Creating the document reference in my other comment below is also done using `fuego.db.doc` instead now.
Does this seem like a good solution?

Reply:
Doing this via the `Fuego` instance is deal. It's nice to have an entry-point that manages all things compatibility. Does 8.x remove this type altogether or just rename it?

Reply:
It doesn't remove it, but since imports are changed from `import * as firebase from` to `import firebase from` it means the version different messes things up. By providing the right firebase in Fuego, this doesn't happen.